internal/clients/team4/utilities.go

```diff
@@ -78,6 +111,18 @@ func buildHistoryInfo(pairs []rules.VariableValuePair) (retInfo judgeHistoryInfo
 	return retInfo, ok
 }
 
+func dump(filename string, format string, v ...interface{}) {
+	//f, err := os.Create(filename)
+	f, err := os.OpenFile(filename, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+	_, err2 := f.WriteString(fmt.Sprintf(format, v...))
+	if err2 != nil {
+		log.Fatal(err2)
+	}
+}
```

Comment:
comment it out so that LH doesn't bash us about it?

Reply:
Thanks!